public/locales/es/health.json

```diff
@@ -1,3 +1,3 @@
 {
-    "heading": "Página de Información de Salud"
+    "pageHeading": "Información de Salud"
```

Comment:
Not translated into Spanish? Waiting on Hillary or Liz?

Reply:
If we're doing Spanish later, remove static content (Spanish) linked issue

Reply:
Intended the Spanish to be listed, I might have missed a commit? I got the Spanish to show before break so that must be it. I'll look on Wednesday.